### DISCUSSION TITLE
Internal Documentation Improvement: add necessary parameters to /quickshop help output.

### DISCUSSION THREAD

**Discussion by mattrose** [QuickShop-Community/QuickShop-Hikari#1499] [Ideas]
I tried to use quickshop to create a shop on a server, and I tried to use `/quickshop create`.   

`/quickshop help` just says "/quickshop create" to create a shop, but it doesn't mention that you need to enter a price.  So when I tried to create a shop I would just get "Invalid argument". I would suggest changing the help text for create to 

`/quickshop create <price>`

**Comment by YuanYuanOwO** [comment]
![{6E0AF6BA-E40A-4dd5-9981-38C831D99BED}](https://github.com/Ghost-chu/QuickShop-Hikari/assets/81153017/804dfd72-8dae-4eff-8580-eb4ca739c2ce)  
When you type a space, it gives you Tab Completion  
https://ghost-chu.github.io/QuickShop-Hikari-Documents/docs/modules/shops/shop-basic#create-a-selling-shop

**Comment by Ghost-chu** [comment]
Create the shop via command is not-recommended
